pkg/kubelet/kubelet.go

```diff
@@ -1041,6 +1057,64 @@ type Kubelet struct {
 	dockerLegacyService dockershim.DockerLegacyService
 }
 
+func initializeServerCertificateManager(kubeClient clientset.Interface, kubeCfg *componentconfig.KubeletConfiguration, nodeName types.NodeName, hostname string) (certificate.Manager, error) {
```

Comment:
they are one type of address that can be returned from the cloud provider `Instances().NodeAddresses()` API (see https://github.com/kubernetes/kubernetes/blob/master/pkg/cloudprovider/providers/aws/aws.go#L931 as an example)